### ISSUE TITLE
nginx behind external load balancer?

### ISSUE THREAD

**Issue by Andrew-Chen-Wang** [pydanny/cookiecutter-django#2548]
I was planning on making a extensive AWS support for EC2 autoscale group OR ECS support for this project and adding a GH workflow action... but I have no clue how to configure nginx to properly run the show in place of traefik since it’s probably best to use ALB in front of everything.

C-Dj is more for one machine in my opinion, especially with celery. Otherwise, it can be highly customized, for sure, especially since IAM roles can be attached to ECS clusters and EC2 instances.

Anyhow, I took a look at the article by msaizer at the bottom of the read the docs regarding nginx, but it uses an image from 4 years ago...

tl;dr: So any help with configuration for nginx behind ALB?

**Comment by Andrew-Chen-Wang** [comment]
Probably not in scope of this. You can see my progress in my fork under the was branch.
